diff -ru gtk-sharp-2.10.2.org/glib/glue/list.c gtk-sharp-2.10.2/glib/glue/list.c
--- gtk-sharp-2.10.2.org/glib/glue/list.c	2012-12-25 10:59:05.203146111 +0000
+++ gtk-sharp-2.10.2/glib/glue/list.c	2012-12-25 11:00:50.383148628 +0000
@@ -20,7 +20,7 @@
  */
 
 
-#include <glib/glist.h>
+#include <glib.h>
 
 /* Forward declarations */
 gpointer gtksharp_list_get_data (GList *l);
diff -ru gtk-sharp-2.10.2.org/glib/glue/slist.c gtk-sharp-2.10.2/glib/glue/slist.c
--- gtk-sharp-2.10.2.org/glib/glue/slist.c	2012-12-25 10:59:05.127146109 +0000
+++ gtk-sharp-2.10.2/glib/glue/slist.c	2012-12-25 11:01:04.155148277 +0000
@@ -20,7 +20,7 @@
  */
 
 
-#include <glib/gslist.h>
+#include <glib.h>
 
 /* Forward declarations */
 gpointer gtksharp_slist_get_data (GSList *l);
diff -ru gtk-sharp-2.10.2.org/glib/glue/thread.c gtk-sharp-2.10.2/glib/glue/thread.c
--- gtk-sharp-2.10.2.org/glib/glue/thread.c	2012-12-25 10:59:05.147146108 +0000
+++ gtk-sharp-2.10.2/glib/glue/thread.c	2012-12-25 11:01:23.279148643 +0000
@@ -20,7 +20,7 @@
  */
 
 
-#include <glib/gthread.h>
+#include <glib.h>
 
 gboolean
 glibsharp_g_thread_supported ()
diff -ru gtk-sharp-2.10.2.org/sample/Actions.cs gtk-sharp-2.10.2/sample/Actions.cs
--- gtk-sharp-2.10.2.org/sample/Actions.cs	2012-12-25 10:59:14.603146276 +0000
+++ gtk-sharp-2.10.2/sample/Actions.cs	2012-12-26 08:46:36.032548890 +0000
@@ -189,7 +189,7 @@
 
 		static void OnActivate (object obj, EventArgs args)
 		{
-			Action action = (Action)obj;
+			Gtk.Action action = (Gtk.Action)obj;
 			Console.WriteLine ("Action {0} (type={1}) activated",
 					   action.Name, action.GetType ().FullName);
 		}
@@ -218,7 +218,7 @@
 
 		static void OnToggleCnp (object obj, EventArgs args)
 		{
-			Action action = (ToggleAction)obj;
+			Gtk.Action action = (ToggleAction)obj;
 			bool sensitive = ((ToggleAction)action).Active;
 			action = group.GetAction ("cut");
 			action.Sensitive = sensitive;
@@ -262,7 +262,7 @@
 			for (int i = 0; i < num; i++) {
 				string name = "DynAction" + i;
 				string label = "Dynamic Action " + i;
-				Action action = new Action (name, label);
+				Gtk.Action action = new Gtk.Action (name, label);
 				dynGroup.Add (action);
 				uim.AddUi (mergeId, "/menubar/DynamicMenu", name,
 					   name, UIManagerItemType.Menuitem, false);
@@ -299,7 +299,7 @@
 
 		static void OnSelect (object obj, EventArgs args)
 		{
-			Action action = (Action) actions[obj];
+			Gtk.Action action = (Gtk.Action) actions[obj];
 			if (action.Tooltip != null)
 				statusbar.Push (0, action.Tooltip);
 		}
diff -ru gtk-sharp-2.10.2.org/sample/GtkDemo/DemoApplicationWindow.cs gtk-sharp-2.10.2/sample/GtkDemo/DemoApplicationWindow.cs
--- gtk-sharp-2.10.2.org/sample/GtkDemo/DemoApplicationWindow.cs	2012-12-25 10:59:14.375146291 +0000
+++ gtk-sharp-2.10.2/sample/GtkDemo/DemoApplicationWindow.cs	2012-12-26 08:42:42.732544622 +0000
@@ -156,7 +156,7 @@
 
 		private void ActionActivated (object sender, EventArgs a)
 		{
-			Action action = sender as Action;
+			Gtk.Action action = sender as Gtk.Action;
 			MessageDialog md;
 
 			md = new MessageDialog (this, DialogFlags.DestroyWithParent,
diff -ru gtk-sharp-2.10.2.org/sample/GtkDemo/DemoUIManager.cs gtk-sharp-2.10.2/sample/GtkDemo/DemoUIManager.cs
--- gtk-sharp-2.10.2.org/sample/GtkDemo/DemoUIManager.cs	2012-12-25 10:59:14.355146264 +0000
+++ gtk-sharp-2.10.2/sample/GtkDemo/DemoUIManager.cs	2012-12-26 08:43:33.388545656 +0000
@@ -132,7 +132,7 @@
 
 		private void ActionActivated (object sender, EventArgs a)
 		{
-			Action action = sender as Action;
+			Gtk.Action action = sender as Gtk.Action;
 			Console.WriteLine ("Action \"{0}\" activated", action.Name);
 		}
 
