Upstream-Status: Inappropriate [Yocto specific]

diff -ur mono-6.14.1.org/mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs mono-6.14.1/mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs
--- mono-6.14.1.org/mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs	2020-01-15 07:46:01.000000000 +0000
+++ mono-6.14.1/mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs	2020-02-08 21:22:11.059830600 +0000
@@ -2818,6 +2818,11 @@ internal override IntPtr DefineCursor(Bitmap bitmap, Bitmap mask, Color cursor_p
 				return IntPtr.Zero;
 			}
 
+			if(width == 0 || height == 0) {
+                          width = bitmap.Width;
+                          height = bitmap.Height;
+                        }
+
 			// Win32 only allows creation cursors of a certain size
 			if ((bitmap.Width != width) || (bitmap.Width != height)) {
 				cursor_bitmap = new Bitmap(bitmap, new Size(width, height));
