--- libgdiplus-2.10.orig/cairo/configure.in	2012-01-23 15:09:19.198498003 -0500
+++ libgdiplus-2.10/cairo/configure.in	2012-01-23 15:08:24.922498003 -0500
@@ -26,23 +26,23 @@
 # libtool shared library version
 
 # Increment if the interface has additions, changes, removals.
-LT_CURRENT=19
+m4_define(lt_current, 19)
 
 # Increment any time the source changes; set to
 # 0 if you increment CURRENT
-LT_REVISION=5
+m4_define(lt_revision, 5)
 
 # Increment if any interfaces have been added; set to 0
 # if any interfaces have been removed. removal has
 # precedence over adding, so set to 0 if both happened.
-LT_AGE=17
+m4_define(lt_age, 17)
 
 dnl ===========================================================================
 
-VERSION_INFO="$LT_CURRENT:$LT_REVISION:$LT_AGE"
+VERSION_INFO="lt_current:lt_revision:lt_age"
 AC_SUBST(VERSION_INFO)
 
-LT_CURRENT_MINUS_AGE=`expr $LT_CURRENT - $LT_AGE`
+LT_CURRENT_MINUS_AGE=`expr lt_current - lt_age`
 AC_SUBST(LT_CURRENT_MINUS_AGE)
 
 CAIRO_VERSION_MAJOR=cairo_version_major()
