diff -ru mono-2.11.org/mono/io-layer/atomic.h mono-2.11/mono/io-layer/atomic.h
--- mono-2.11.org/mono/io-layer/atomic.h	2012-03-21 17:39:19.000000000 +0000
+++ mono-2.11/mono/io-layer/atomic.h	2012-12-23 19:48:18.680623763 +0000
@@ -752,7 +752,7 @@
 
 static inline gint32 InterlockedCompareExchange(volatile gint32 *dest, gint32 exch, gint32 comp)
 {
-#if defined(__ARM_ARCH_6__) || defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
+#if defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
 	gint32 ret, tmp;
 	__asm__ __volatile__ (	"1:\n"
 				"dmb\n"
@@ -792,7 +792,7 @@
 
 static inline gpointer InterlockedCompareExchangePointer(volatile gpointer *dest, gpointer exch, gpointer comp)
 {
-#if defined(__ARM_ARCH_6__) || defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
+#if defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
 	gpointer ret, tmp;
 	__asm__ __volatile__ (
 				"dmb\n"
@@ -833,7 +833,7 @@
 
 static inline gint32 InterlockedIncrement(volatile gint32 *dest)
 {
-#if defined(__ARM_ARCH_6__) || defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
+#if defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
 	gint32 ret, flag;
 	__asm__ __volatile__ (
 				"dmb\n"
@@ -869,7 +869,7 @@
 
 static inline gint32 InterlockedDecrement(volatile gint32 *dest)
 {
-#if defined(__ARM_ARCH_6__) || defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
+#if defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
 	gint32 ret, flag;
 	__asm__ __volatile__ (
 				"dmb\n"
@@ -905,7 +905,7 @@
 
 static inline gint32 InterlockedExchange(volatile gint32 *dest, gint32 exch)
 {
-#if defined(__ARM_ARCH_6__) || defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
+#if defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
 	gint32 ret, flag;
 	__asm__ __volatile__ (
 				  "dmb\n"
@@ -932,7 +932,7 @@
 
 static inline gpointer InterlockedExchangePointer(volatile gpointer *dest, gpointer exch)
 {
-#if defined(__ARM_ARCH_6__) || defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
+#if defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
 	gpointer ret, flag;
 	__asm__ __volatile__ (
 				  "dmb\n"
@@ -959,7 +959,7 @@
 
 static inline gint32 InterlockedExchangeAdd(volatile gint32 *dest, gint32 add)
 {
-#if defined(__ARM_ARCH_6__) || defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
+#if defined(__ARM_ARCH_7A__) || defined(__ARM_ARCH_7__)
 	gint32 ret, tmp, flag;
 	__asm__ __volatile__ (
 				"dmb\n"
