diff -ur mono-6.8.0.96.org/mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs mono-6.8.0.96/mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs
--- mono-6.8.0.96.org/mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs	2020-01-15 07:46:01.000000000 +0000
+++ mono-6.8.0.96/mcs/class/System.Windows.Forms/System.Windows.Forms/XplatUIX11.cs	2020-02-08 21:22:11.059830600 +0000
@@ -3091,6 +3091,11 @@
 				return IntPtr.Zero;
 			}
 
+			if(width == 0 || height == 0) {
+                          width = bitmap.Width;
+                          height = bitmap.Height;
+                        }
+
 			// Win32 only allows creation cursors of a certain size
 			if ((bitmap.Width != width) || (bitmap.Width != height)) {
 				cursor_bitmap = new Bitmap(bitmap, new Size(width, height));
