diff -ur git.org/data/config.in git/data/config.in
--- git.org/data/config.in	2014-05-30 20:35:17.062695645 +0100
+++ git/data/config.in	2014-05-30 21:02:10.338699860 +0100
@@ -27,8 +27,8 @@
 		<dllentry dll="__Internal" name="MoveMemory" target="mono_win32_compat_MoveMemory"/>
 		<dllentry dll="__Internal" name="ZeroMemory" target="mono_win32_compat_ZeroMemory"/>
 	</dllmap>
-	<dllmap dll="gdiplus" target="@prefix@/lib/libgdiplus@libsuffix@" os="!windows"/>
-	<dllmap dll="gdiplus.dll" target="@prefix@/lib/libgdiplus@libsuffix@"  os="!windows"/>
+	<dllmap dll="gdiplus" target="libgdiplus.so.0" os="!windows"/>
+	<dllmap dll="gdiplus.dll" target="libgdiplus.so.0"  os="!windows"/>
 	<dllmap dll="gdi32" target="@prefix@/lib/libgdiplus@libsuffix@" os="!windows"/>
 	<dllmap dll="gdi32.dll" target="@prefix@/lib/libgdiplus@libsuffix@" os="!windows"/>
 </configuration>
