diff -ur mono-5.16.0.179.orig/data/config.in mono-5.16.0.179/data/config.in
--- mono-5.16.0.179.orig/data/config.in	2018-10-05 03:46:00.000000000 -0400
+++ mono-5.16.0.179/data/config.in	2018-11-08 03:49:21.565183148 -0500
@@ -10,14 +10,14 @@
 	<dllmap dll="i:odbc32.dll" target="libiodbc.dylib" os="osx"/>
 	<dllmap dll="oci" target="libclntsh@libsuffix@" os="!windows"/>
 	<dllmap dll="db2cli" target="libdb2_36@libsuffix@" os="!windows"/>
-	<dllmap dll="MonoPosixHelper" target="$mono_libdir/libMonoPosixHelper@libsuffix@" os="!windows" />
+	<dllmap dll="MonoPosixHelper" target="libMonoPosixHelper.so" os="!windows" />
 	<dllmap dll="System.Native" target="$mono_libdir/libmono-system-native@libsuffix@" os="!windows" />
 	<dllmap dll="libmono-btls-shared" target="$mono_libdir/libmono-btls-shared@libsuffix@" os="!windows" />
 	<dllmap dll="i:msvcrt" target="@LIBC@" os="!windows"/>
 	<dllmap dll="i:msvcrt.dll" target="@LIBC@" os="!windows"/>
 	<dllmap dll="sqlite" target="@SQLITE@" os="!windows"/>
 	<dllmap dll="sqlite3" target="@SQLITE3@" os="!windows"/>
-	<dllmap dll="libX11" target="@X11@" os="!windows" />
+	<dllmap dll="libX11" target="libX11.so.6" os="!windows" />
 	<dllmap dll="libgdk-x11-2.0" target="@GDKX11@" os="!windows"/>
 	<dllmap dll="libgdk_pixbuf-2.0" target="libgdk_pixbuf-2.0.so.0" os="!windows"/>
 	<dllmap dll="libgtk-x11-2.0" target="@GTKX11@" os="!windows"/>
@@ -37,8 +37,8 @@
 		<dllentry dll="__Internal" name="MoveMemory" target="mono_win32_compat_MoveMemory"/>
 		<dllentry dll="__Internal" name="ZeroMemory" target="mono_win32_compat_ZeroMemory"/>
 	</dllmap>
-	<dllmap dll="gdiplus" target="@libgdiplus_install_loc@" os="!windows"/>
-	<dllmap dll="gdiplus.dll" target="@libgdiplus_install_loc@"  os="!windows"/>
+	<dllmap dll="gdiplus" target="libgdiplus.so.0" os="!windows"/>
+	<dllmap dll="gdiplus.dll" target="libgdiplus.so.0"  os="!windows"/>
 	<dllmap dll="gdi32" target="@libgdiplus_install_loc@" os="!windows"/>
 	<dllmap dll="gdi32.dll" target="@libgdiplus_install_loc@" os="!windows"/>
 </configuration>
